idc_index/index.py

```diff
@@ -58,20 +58,19 @@ def client(cls) -> IDCClient:
         return cls._client
 
     def __init__(self):
+        # Read main index file
         file_path = idc_index_data.IDC_INDEX_PARQUET_FILEPATH
-
-        # Read index file
         logger.debug(f"Reading index file v{idc_index_data.__version__}")
         self.index = pd.read_parquet(file_path)
         # self.index = self.index.astype(str).replace("nan", "")
         self.index["series_size_MB"] = self.index["series_size_MB"].astype(float)
         self.collection_summary = self.index.groupby("collection_id").agg(
             {"Modality": pd.Series.unique, "series_size_MB": "sum"}
         )
+        self.indices_overview = None
```

Comment:
Why not initialize this variable by calling `_get_latest_idc_index_data_release_assets`? Or at least it should contain an entry corresponding to the main index, which will always be initialized.

Reply:
Yes, you are right. We could also simply call the list_indices function here. So in case download of the main index is going wrong, we can show that to the user.